.circleci/config.yml

```diff
@@ -11,7 +11,8 @@ jobs:
       - run:
           name: Install python dependencies
           command: |
-            pip install poetry==1.1.7
+            sudo pip uninstall --yes poetry
```

Comment:
Is this needed for this container?

Reply:
Yes, otherwise it looks like it uses the wrong version of poetry:
https://app.circleci.com/pipelines/github/lace/vg/1548/workflows/250be52e-896f-44ba-965a-399fb982b81b/jobs/3452

Reply:
Does it work without sudo?

Reply:
no.  https://app.circleci.com/pipelines/github/lace/vg/1557/workflows/d94574be-3aa2-495b-a69a-03f522c231bd/jobs/3461